src/System.Drawing.Common/src/System/Drawing/Graphics.Windows.cs

```diff
@@ -860,29 +860,23 @@ public void DrawBezier(Pen pen, Point pt1, Point pt2, Point pt3, Point pt4)
         /// <summary>
         /// Draws a series of cubic Bezier curves from an array of points.
         /// </summary>
-        public void DrawBeziers(Pen pen, Point[] points)
+        public unsafe void DrawBeziers(Pen pen, Point[] points)
         {
             if (pen == null)
-            {
                 throw new ArgumentNullException(nameof(pen));
-            }
 
             if (points == null)
-            {
                 throw new ArgumentNullException(nameof(points));
-            }
 
-            IntPtr buf = SafeNativeMethods.Gdip.ConvertPointToMemory(points);
-            try
+            fixed (Point* p = points)
```

Comment:
It looks like ConvertPointToMemory is doing some complicated marshaling.  That's all unnecessary, and Point is blittable and matching the corresponding native types in layout?

Reply:
Yes, Point matches the native type and is blittable. I'm not sure how they stumbled into that loop originally.

Reply:
The same pattern is used at about 60 places. It would be nice to get all of them fixed up.

Reply:
> The same pattern is used at about 60 places. 

Yep. Following up on it right now. Just wanted to get an initial change up to show the general pattern of what I'm doing.